test/fixtures/merge_sub_file.yaml

```diff
@@ -0,0 +1,110 @@
+subs:
+-
+  hostname: .*
+  type: Hypervisor
+  skip_density: true
+  facts:
+    -
+      name: hypervisor::cluster
+      value: example_cluster
+      matcher: regexp
+  sub:
+    rhel:
+      - name = "Red Hat Enterprise Linux for Virtual Datacenters, Premium"
```

Comment:
`name: Red Hat Enterprise Linux for Virtual Datacenters, Premium`, not `name = "Red Hat Enterprise Linux for Virtual Datacenters, Premium"`